src/components/Book.js

```diff
@@ -2,27 +2,61 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBookFromApi } from '../redux/books/booksSlice';
+import ProgressBar from './ProgressShow';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   return (
-    <>
-      <p key={book.id}>
-        <strong>{book.title}</strong>
-        {' '}
-        <span>by</span>
-        {' '}
-        <strong>{book.author}</strong>
-      </p>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(deleteBookFromApi(book.id));
-        }}
-      >
-        Delete
-      </button>
-    </>
+    <div className="book-Card">
+      <div key={book.id} className="book-info">
+        <p className="class-">{book.category}</p>
+        <p className="Title">{book.title}</p>
+        <p className="Author">{book.author}</p>
+        <ul>
+          <li>
+            <button className="Comments" type="button">
+              Comments
+            </button>
+          </li>
+          <div className="level-one" />
+          <li>
+            <button
+              className="Remove"
+              type="button"
+              onClick={() => {
+                dispatch(deleteBookFromApi(book.id));
+              }}
+            >
+              Remove
+            </button>
+          </li>
+          <div className="level-one" />
+          <li>
+            <button className="Edit" type="button">
+              Edit
+            </button>
+          </li>
+        </ul>
+      </div>
+      <div className="item-progress">
+        <ProgressBar />
+        <div className="task-progress">
+          <p className="Complete-Task">
+            {Math.floor(Math.random() * (99 - 0)) + 0}
+            <span>%</span>
+          </p>
+          <p className="Completed">Completed</p>
+        </div>
+      </div>
+      <div className="level-two" />
+      <div className="levels">
+        <p className="main-levels">CURRENT CHAPTER</p>
+        <p className="main-task">Chapter 12</p>
+        <button type="button" className="Update-progress">
+          UPDATE PROGRESS
+        </button>
+      </div>
```

Comment:
- [ ] Kindly adjust the space between the items here to match the template. Your design looks a bit cluttered. ✍️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/230080711-940df652-1e77-4b38-acdf-89404b7014eb.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/230080702-cf63a76f-1bc3-4740-af56-667cd142c53f.png" width="300">|